src/components/app/data/services/bffs.js

```diff
@@ -0,0 +1,34 @@
+import { getConfig } from '@edx/frontend-platform/config';
+import { logError } from '@edx/frontend-platform/logging';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { camelCaseObject } from '@edx/frontend-platform/utils';
+
+export const learnerDashboardBFFResponse = {
+  enterpriseCustomerUserSubsidies: {
+    subscriptions: {
+      customerAgreement: {},
+      subscriptionLicenses: [],
+      subscriptionLicensesByStatus: {},
+    },
+  },
+  enterpriseCourseEnrollments: [],
+  errors: [],
+  warnings: [],
+};
```

Comment:
[suggestion] could create a couple helper abstraction variables for future reusability of the common route data, e.g.

```javascript
export const baseLearnerBFFResponse = {
  enterpriseCustomerUserSubsidies: {
    subscriptions: {
      customerAgreement: {},
      subscriptionLicenses: [],
      subscriptionLicensesByStatus: {},
    },
  },
  errors: [],
  warnings: [],
}

export const learnerDashboardBFFResponse = {
  ...baseLearnerBffResponse,
  enterpriseCourseEnrollments: [],
}
```